server.js

```diff
@@ -35,6 +42,10 @@ socket.on('message', function (msg, rinfo) {
         return;
     }
 
+    if(use_nemesis) {
+    	exec("nemesis udp -c 1 -x 10001 -y " + rinfo.port + " -P unifi-server.packet -S " + controller_ip + " -D " + rinfo.address);
```

Comment:
i'm not familiar with `nemesis` so this question may be misguided. Should the code `return` here? This will fork `nemesis` for *every* incoming connection - is that the goal?

Is `nemesis` going to run forever, or just for a *small* amount of time?

Reply:
This code will run nemesis for each incoming message, which is equal to "request" variable (same as before).
Nemesis will send one spoofed packet and exit.
This is very rudimentary way to achieve this, because I didn't find solution for nodejs to send raw IP-packets.

If you have more questinons fell free to ask me. :)

Reply:
@rick51231 do you just let server portion run all the time? If I don't leave it running, the next time I open the app (Android) it goes back to remotely connected...

Reply:
Of course it has to run all the time to serve the connection requests.
I have it running on a RasPi and start it with
sudo node ./server.js packet.json nohup &